### DISCUSSION TITLE
EU wall light switch that still uses ESP?

### DISCUSSION THREAD

**Discussion by evlo** [arendst/Tasmota#16343] [Q&A] [answered]
As now most tuya switches use tuya chips instead of ESP shopping for some switch that I can really control gets hard.

Currently I have zigbee version of these https://www.aliexpress.com/item/1005002777322012.html but in the end even thought all look same externally, they have different options and mainly it waits for doubleclick so there is delay.

So I'm looking for EU wall light switch that still uses ESP chips, does have hw clicky buttons, no touch, would be nice to have controllable very dim white light for locating the switch at night.
And as sad as it is to write this, I require it not to do any beeping on press :)

I only found Athom, but they don't have clicky one with shipping from EU and customs are now quite crazy for stuff outside eu, so i would like some alternatives if you know some.

Since would perfect if there is zigbee battery version of the switch with the same design.

For example this https://www.aliexpress.com/item/4000242679288.html or this https://www.aliexpress.com/item/1005004428978918.html looks nice, no note about switch away from esp here https://templates.blakadder.com/girier_EK03.html and https://templates.blakadder.com/lerlink_X802A-L.html but can someone confirm that? It seems really rare to cam across device that is not using tuya chips.

**Comment by Jason2866** [comment, marked as answer]
All our knowledge about what has been working at some day is in our official template site.
https://templates.blakadder.com/

**Reply by evlo** [reply]
but that does not help with what is working, since what i described and what has been noted on some of the switches on the link both you and I posted.

**Reply by sfromis** [reply]
Then you are asking for something "we" don't know. Even assuming that someone bought a device recently, and found it to still be using an ESP chip, it is still not at all unlikely that current batch will be a revised design with different selection of internals.

**Reply by evlo** [reply]
So let me ask different way - are there other   onpanies like Athome that basically guarantee esp chip?

**Reply by sfromis** [reply]
Again, what "we" know is in the templates repository. https://templates.blakadder.com/preflashed.html

**Comment by Jason2866** [comment]
Once again all we know is listed at https://templates.blakadder.com/

**Comment by barbudor** [comment]
If you want something 100% sure, get good old mechanical switches and hook a Shelly or a Sonoff mini in the wall

I've done that because none of what is available on Ali has WAF for either the touch or the look.
Legrand and other EU manufacturers have great mechanical switches with wide choice of look and colors

**Reply by evlo** [reply]
Thought about that, but then if I use other control than button itself  it will be in down position when light is on and vice versa, right?

**Reply by barbudor** [reply]
Wouldn't that be true if it is built-in ?

When I say mechanical I didn't necesseraly meant rocker switch. It can be a momentary push button too. No possition hassle

**Reply by sfromis** [reply]
You could create rules (or other automation) to make whatever dependence between local actions (physical switch) and other controls. To avoid "wrong" physical position, you could also use physical switches with spring-loaded momentary push, instead of locking toggle. Some wall switches supports both, with the installation of a suitable spring.

**Reply by evlo** [reply]
> Wouldn't that be true if it is built-in ?
> 
> When I say mechanical I didn't necesseraly meant rocker switch. It can be a momentary push button too. No possition hassle

Hmm, i did honestly not know such thing even existed, i found only abb A87345 that works like that, at least so far for me it would mean diggin up the wall as lot of times there is not enough space behind the switch.

With shelly, if it gets disconnected from zigbee/wifi will the relay still work when it receives signal from the button? (If there is actual light connected to shelly and not using button press to trigger some routine etc.)

**Reply by barbudor** [reply]
Shelly or not Shelly, this is a problem of behavior of Tasmota. There have been some improvements recently on local operations when Tamsota doesn't have anymore Wifi connection.

Using a standard pushbutton + Shelly is no different than getting an all-in-one chinese with nuilt-in ESP
You just have more choice on the switch quality (and to be franck may be a little more difficulty to fit the whole in the wall)

**Reply by evlo** [reply]
well the chinese, at least, zigbee switches, that go behind socket/switch don't last long from my experience, at least zigbee ones, i did not try wifi as it si now almost never esp based these days, but zigbee ones work when connection to coordinator is lost, so i kinda thought that is hw thing :/ Even with small chinese one I needed to take it out of already small case to fit it, i pretty much expect i would need to double depth of each socket box.

Something like athom, that is guaranteed esp, but more designs would be ideal, but reality is reality :)
I think it is either shelly or some small "development" board

I wonder what tuya long term plan is as they really really want you in their system, even thought, from my point of view, it is only more expensive for them to support device infrastructure after sale of the hw, maybe it is just so they can sell subscription service after few years. Or maybe it is cheaper for them to create their own chip then to buy from esperiff.

**Reply by evlo** [reply]
Shelly sells "WALL SWITCH 2" that does have mount for the shelly thing behind it, so it should solve need for the deep wall socket. They also have 4 button version - but they do not have any 4 relay thing behind it, only "scene thing". I guess it can work with shelly when there is no wifi connection and for smart automations i can use whatever webhooks it does have.

**Reply by barbudor** [reply]
Interresting, I was not aware.

The 4 buttons goes with the I4 which is only to trigger remote devices. Like two-ways... Or switching smart-bulbs

**Reply by evlo** [reply]
sadly it is the case, max 2 for it to still work offline, and offline + instant response is required for the device to be home accepted :)

**Reply by evlo** [reply]
Another thought - with 4 button holder one could use one shelly for "scene" trigger and another to control real lights using relays as maybe with the holder you could fit 2 units in the small space behind the switch.

**Reply by barbudor** [reply]
If you have the room for 2 shellies then you can use 2 x Shelly plus 2PM
Apparently the relay holder is only compatible with the Plus serie
